test/arcade/potts/agent/cell/PottsCellStemTest.java

```diff
@@ -0,0 +1,268 @@
+    
+    static int cellDivisions = randomIntBetween(1, 100);
+    
+    static double cellCriticalVolume = randomDoubleBetween(10, 100);
+    
+    static double cellCriticalHeight = randomDoubleBetween(10, 100);
+    
+    static State cellState = State.QUIESCENT;
+    
+    static PottsCell cellDefault;
+    
+    static PottsCell cellWithRegions;
+    
+    static PottsCell cellWithoutRegions;
+    
+    static EnumSet<Region> regionList;
+    
+    static MiniBox parametersMock;
+    
+    @BeforeClass
+    public static void setupMocks() {
+        parametersMock = mock(MiniBox.class);
+        locationMock = mock(PottsLocation.class);
+        
+        regionList = EnumSet.of(Region.DEFAULT, Region.NUCLEUS);
+        when(locationMock.getRegions()).thenReturn(regionList);
+        
+        Answer<Double> answer = invocation -> {
+            Double value1 = invocation.getArgument(0);
+            Double value2 = invocation.getArgument(1);
+            return value1 * value2;
+        };
+        when(((PottsLocation) locationMock).convertSurface(anyDouble(), anyDouble())).thenAnswer(answer);
+        
+        locationRegionVolumes = new EnumMap<>(Region.class);
+        locationRegionHeights = new EnumMap<>(Region.class);
+        locationRegionSurfaces = new EnumMap<>(Region.class);
+        
+        // Random volumes and surfaces for regions.
+        for (Region region : regionList) {
+            locationRegionVolumes.put(region, randomIntBetween(10, 20));
+            locationRegionHeights.put(region, randomIntBetween(10, 20));
+            locationRegionSurfaces.put(region, randomIntBetween(10, 20));
+            
+            when(locationMock.getVolume(region)).thenReturn((double) locationRegionVolumes.get(region));
+            when(locationMock.getHeight(region)).thenReturn((double) locationRegionHeights.get(region));
+            when(locationMock.getSurface(region)).thenReturn((double) locationRegionSurfaces.get(region));
+            
+            locationVolume += locationRegionVolumes.get(region);
+            locationHeight += locationRegionHeights.get(region);
+            locationSurface += locationRegionSurfaces.get(region);
+        }
+        
+        when(locationMock.getVolume()).thenReturn((double) locationVolume);
+        when(locationMock.getHeight()).thenReturn((double) locationHeight);
+        when(locationMock.getSurface()).thenReturn((double) locationSurface);
+        
+        // Region criticals.
+        criticalVolumesRegionMock = new EnumMap<>(Region.class);
+        criticalHeightsRegionMock = new EnumMap<>(Region.class);
+        for (Region region : regionList) {
+            criticalVolumesRegionMock.put(region, (double) locationRegionVolumes.get(region));
+            criticalHeightsRegionMock.put(region, (double) locationRegionHeights.get(region));
+        }
+        
+        cellDefault = new PottsCellStem(cellID, cellParent, cellPop, cellState, cellAge, cellDivisions,
```

Comment:
Don't need any of these cells because they are not used for any of the tests in this class